docs-ref-mapping/reference.yml

```diff
@@ -879,10 +884,15 @@
   - name: Monitor
     uid: azure.nodejs.sdk.landingpage.services.monitor
     href: ~/docs-ref-services/latest/monitor.md
+    landingPageType: Service
```

Comment:
```suggestion
```

Reply:
You have a `href`, `href` and `landingPageType: Service` are mutually exclusive.